src/components/version-context-switcher/index.tsx

```diff
@@ -36,10 +36,22 @@ const VersionContextSwitcher = ({
     }
   }
 
+  /**
+   * TODO: hopefully this is temporary. Because Sentinel does not have a
```

Comment:
Random note: I'm not sure of the path to make this not temporary - I don't _think_ there are any plans to make an icon for Sentinel anytime soon... But this issue is definitely outside the scope of this PR, so I've added an Asana ticket 🎟️ : [[discuss] Sentinel icon](https://app.asana.com/0/1201010428539925/1201948872798561/f)